lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -1103,10 +1103,15 @@ def add_installments(xml, options)
       end
 
       def add_threeds_services(xml, options)
-        xml.tag! 'payerAuthEnrollService', { 'run' => 'true' } if options[:payer_auth_enroll_service]
+        if options[:payer_auth_enroll_service]
+          xml.tag! 'payerAuthEnrollService', { 'run' => 'true' } do
+            xml.tag! 'MCC', options[:mcc] if options[:mcc]
+          end
+        end
+
         if options[:payer_auth_validate_service]
           xml.tag! 'payerAuthValidateService', { 'run' => 'true' } do
-            xml.tag! 'signedPARes', options[:pares]
+            xml.tag! 'signedPARes', options[:pares] if options[:pares]
```

Comment:
Any particular reason to add this? I think it should be reliably included in `options` based on the way we communicate these details from the Spreedly side